.changes/unreleased/Added-20241201-115758.yaml

```diff
@@ -0,0 +1,3 @@
+kind: Added
+body: 'commit {create, amend, split}branch create: Adds --no-verify flag to bypass pre-commit and commit-msg hooks.'
```

Comment:
Heh, thanks for trying.
No hard rules on this—just something that's obvious to a reader.

```suggestion
body: 'Add --no-verify flag to bypass pre-commit and commit-msg hooks to commands that create new commits.'
```